src/main/java/com/epam/esm/exception/EntityAlreadyExistsException.java

```diff
@@ -0,0 +1,7 @@
+package com.epam.esm.exception;
+
+public class EntityAlreadyExistsException extends Exception {
```

Comment:
Why all these exceptions extend `Exception` class?

Reply:
Since we are not going to catch the exceptions and handle them somehow in catch{} section, it should extend RuntimeExceprion class. This way I implemented it